internal/tokens/fftokens/fftokens.go

```diff
@@ -643,13 +662,68 @@ func (ft *FFTokens) ActivateTokenPool(ctx context.Context, nsOpID string, pool *
 	return false, nil
 }
 
-func (ft *FFTokens) MintTokens(ctx context.Context, nsOpID string, poolLocator string, mint *core.TokenTransfer) error {
+func (ft *FFTokens) prepareABI(ctx context.Context, methods []*fftypes.FFIMethod) ([]*abi.Entry, error) {
+	abiMethods := make([]*abi.Entry, len(methods))
```

Comment:
This line confirms for me the comment earlier - that the `abi` field of the payload passed to the Token connector is _not_ the full ABI, but rather the array of ABI-formatted Methods.

... all to say, I feel quite strongly we need to rename `methods` to `ffi` in the payload to the token connector (or rename _both_ to `abiMethods`/`ffiMethods`)

Reply:
Or should it be called `methods` in both cases (and the `format` param tells you how to decode it)?

Reply:
Since it's not _really_ an `abi` or `ffi`, it's always a list of methods encoded in `abi` or `ffi` format.